packages/webapp/src/routes/+layout.ts

```
@@ -10,26 +10,42 @@ import pkg from 'lodash';
-	const activeNetworkRef = writable<string>('');
-	const activeOrderbookRef = writable<string>('');
-	const activeOrderbook = derived(
+	let settingsJson: ConfigSource | undefined;
+	let errorMessage: string | undefined;
+
+	try {
+		const response = await fetch(
+			'https://raw.githubusercontent.com/rainlanguage/rain.strategies/refs/heads/main/settings.json'
+		);
+		if (!response.ok) {
+			throw new Error('Error status: ' + response.status.toString());
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)